docs/publish.sh

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 echo 'cloning'
-git clone git@github.com:envoy-mobile/envoy-mobile.github.io "$CHECKOUT_DIR"
+git clone git@github.com:envoy-mobile/envoymobile.io "$CHECKOUT_DIR"
```

Comment:
Does this repository exist?
```
Cloning into 'envoymobile.io'...
ERROR: Repository not found.
fatal: Could not read from remote repository.
```

Reply:
No, this was renamed by mistake. Good catch! Fixed in 5a801151